tests/x11/nis_server.pm

```diff
@@ -68,7 +68,7 @@ sub nis_server_configuration {
     assert_screen 'nis-server-server-maps-setup-finished';
     send_key $cmd{next};
     # NIS Server Query Hosts
-    assert_screen 'nis-server-query-hosts-setup';
+    apply_workaround_bsc1204176('nis-server-query-hosts-setup') if (is_sle('>=15-SP4'));
```

Comment:
What will be there if VERSION less than 15-SP4?

Reply:
It is related with bsc1204176. It doesn't have this problem except 15sp5.

Reply:
Ok, so this should need workaround for bsc1204176.